wiki/Accuracy/en.md

```diff
@@ -99,5 +128,8 @@ Spin is only for osu!Standard
 
 **Notes**
 
--   The above four values are not saved and will disappear after you close osu!. You can only see them again by watching a saved replay.
--   Due to the way the Double Time and Half Time mods are implemented, the error and unstable rate values will be multiplied by the same factor as the song. To get the true values when playing DT, divide the results by 1.5. Similarly, multiply the results by 1.33 when playing HT.
+-   The above four values are not saved and will disappear after closing _osu!_.
+    To see them again, watch an exported replay.
+-   Due to the way the Double Time (DT) and Half Time (HT) mods are implemented, the error and unstable rate values will be multiplied by the same factor as the song.
```

Comment:
I'd reverse the full term and the abbreviation.
`Due to the way the DT (Double Time) and HT (Half Time) mods are implemented, the error and unstable rate values will be multiplied by the same factor as the song`